subprojects/docs/src/docs/userguide/dep-man/04-modeling-features/artifact_transforms.adoc

```diff
@@ -95,6 +95,7 @@ include::sample[dir="snippets/dependencyManagement/artifactTransforms-minify/gro
 <2> All JAR files are not minified
 <3> Request `minified=true` on all resolvable configurations
 <4> Add the dependencies which will be transformed
+<5> Add task to execute the artifact transformation
```

Comment:
"execute" here makes it sound like resolveRuntimeClasspath is doing the work of the artifact transform.

What about...
```suggestion
<5> Add task that requires the transformed artifacts
```